lib/cypress/measure_evaluator.rb

```diff
@@ -67,13 +67,13 @@ def wait_for_calculations
       continuing = true
       while continuing
         sleep(2)
-        continuing = ProductTest.where(state: 'pending', product: product).count > 0
+        continuing = ProductTest.where(state: 'pending', product: product).by_updated_at.count > 0
       end
       @logger.info 'done'
     end
 
     def upload_all_cat3s
-      upload_cat3s(Product.where(name: 'MeasureEvaluationProduct').first.product_tests)
+      upload_cat3s(Product.where(name: 'MeasureEvaluationProduct').by_updated_at.first.product_tests)
```

Comment:
by_updated_at.first is kinda wonky. Shouldn't that be a scope called "most_recent' or something like that? I see a bunch of places where we're doing something like that